attachment_helpers.py

```diff
@@ -32,11 +32,22 @@ def get_download_url(video_key, recording_method_is_pipe):
         bucket = settings.S3_BUCKET_NAME
 
     try:
-        url = S3_CLIENT.generate_presigned_url(
-            "get_object",
-            Params={"Bucket": bucket, "Key": video_key},
-            ExpiresIn=600,
-        )
+        if set_attachment_header:
+            url = S3_CLIENT.generate_presigned_url(
+                "get_object",
+                Params={
+                    "Bucket": bucket,
+                    "Key": video_key,
+                    "ResponseContentDisposition": "attachment",
+                },
+                ExpiresIn=600,
+            )
```

Comment:
600 seconds? Or minutes?

Reply:
Also, it looks like the only difference between the if and else is the params.  It might be cleaner if you pulled out the params dictionary and had only it in the conditional.  This is a code style note, feel free to ignore.

Reply:
> 600 seconds? Or minutes?

@okaycj 600 seconds, i.e. 10 minutes. I kept this value the same. Do you think it should change?